### PR TITLE
get_applicable_library_items_by_datapoints method VerificationLibrary 

### PR DESCRIPTION
**Pull request by yunjoonjung-PNNL**
This PR addresses the development of the ```get_applicable_library_items_by_datapoints``` method in the VerificationLibrary class.

### CODE REVIEW

**Comment by yunjoonjung-PNNL** [comment]
@leijerry888 @yanchenpnnl 

I focused on correcting the inconsistent data point nomenclature uses across verification items instead of trying to fix all the data point nomenclature based on Yan's nomenclature outcome from ```Control_Library.xlsx```. I'll go another round of fixing it once all the team members agree on the nomenclature.

**Comment by lymereJ** [comment]
> @leijerry888 @yanchenpnnl
> 
> I focused on correcting the inconsistent data point nomenclature uses across verification items instead of trying to fix all the data point nomenclature based on Yan's nomenclature outcome from `Control_Library.xlsx`. I'll go another round of fixing it once all the team members agree on the nomenclature.

Should we have a unit test to enforce proper naming following the nomenclature?

File: src/api/verification_library.py [hunk 1]
```diff
@@ -100,3 +100,41 @@ def get_library_item(self, item_name: str) -> Dict:
...
+        Returns:
+            Dict with keys being the library item names and values being the required datapoints for the corresponding keys.
+        """
+
+        # check `datapoints` type
+        if not isinstance(datapoints, List):
+            logging.error(
+                f"datapoints' type must be List. It can't be {type(datapoints)}."
+            )
+            return None
```
Comment (lymereJ) [inline src/api/verification_library.py:154]: Same comment as other PRs about empty list.

Comment (yunjoonjung-PNNL) [reply src/api/verification_library.py:154]: Addressed.

**Comment by yunjoonjung-PNNL** [comment]
> 

I believe this is a really good idea! Based on @leijerry888 and @yanchenpnnl's opinions, I'll update this PR.

**Review by leijerry888 — APPROVED** [review]

**Pull request merged**